tests/integration/webui_helper.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import json
+import logging
+import time
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+SUBSCRIBER_CONFIG = {
+    "UeId": "PLACEHOLDER",
+    "plmnId": "20801",
+    "opc": "981d464c7c52eb6e5036234984ad0bcf",
+    "key": "5122250214c33e723a5dd523fc145fc0",
+    "sequenceNumber": "16f3b3f70fc2",
+}
+DEVICE_GROUP_CONFIG = {
+    "imsis": [],
+    "site-info": "demo",
+    "ip-domain-name": "pool1",
+    "ip-domain-expanded": {
+        "dnn": "internet",
+        "ue-ip-pool": "172.250.1.0/16",
+        "dns-primary": "8.8.8.8",
+        "mtu": 1460,
+        "ue-dnn-qos": {
+            "dnn-mbr-uplink": 200000000,
+            "dnn-mbr-downlink": 200000000,
+            "traffic-class": {"name": "platinum", "arp": 6, "pdb": 300, "pelr": 6, "qci": 8},
+        },
+    },
+}
+NETWORK_SLICE_CONFIG = {
+    "slice-id": {"sst": "1", "sd": "010203"},
+    "site-device-group": [],
+    "site-info": {
+        "site-name": "demo",
+        "plmn": {"mcc": "208", "mnc": "93"},
+        "gNodeBs": [{"name": "demo-gnb1", "tac": 1}],
+        "upf": {"upf-name": "upf", "upf-port": "8805"},
+    },
+}
+
+
+class WebUI:
+    def __init__(self, webui_ip: str) -> None:
+        """Constructor.
+
+        Args:
+            webui_ip (str): IP address of the WebUI application unit
+        """
+        self.webui_ip = webui_ip
+
+    def create_subscriber(self, imsi: str) -> None:
+        """Creates a subscriber.
+
+        Args:
+            imsi (str): Subscriber's IMSI
+        """
+        SUBSCRIBER_CONFIG["UeId"] = imsi
```

Comment:
Are we intentionally persisting the modifications to the `SUBSCRIBER_CONFIG` data? Should we?
```suggestion
    subscriber_config = SUBSCRIBER_CONFIG.copy()
    subscriber_config["UeId"] = imsi
```

Reply:
Same in the other methods.